src/js/eva.js

```diff
@@ -490,10 +490,8 @@ Eva.prototype = {
             var pubmedId = obj.html();
             obj.html('<p>Attempting to retrieve publication information for PubMed ID <a class="external publication" href="http://europepmc.org/abstract/MED/' + pubmedId + '">' + pubmedId + '...</p>');
             if(pubmedId && pubmedId != '-') {
-                var id_type = 'PubMed';
-
                 var host = METADATA_HOST.replace("/eva/webservices/rest", "");
-                var url = host + '/ega/publications/get/paper/' + id_type + '/' + pubmedId;
+                var url = host + '/europepmc/webservices/rest/search?query=ext_id:' + pubmedId + ' src:med&format=json';
```

Comment:
This was already pushed [in another PR](https://github.com/EBIvariation/eva-web/pull/23). Could you please `git rebase develop`?